ui/app/components/secret-edit.js

```diff
@@ -156,6 +156,22 @@ export default Component.extend(FocusOnInsertMixin, WithNavToNearestAncestor, {
     return this.secretDataIsAdvanced || this.preferAdvancedEdit;
   }),
 
+  isWriteWithoutRead: computed(
+    'model.{failedServerRead,selectedVersion.failedServerRead}',
+    'isV2',
+    function() {
+      // if the version couldn't be read from the server
+      if (this.isV2 && this.model.selectedVersion.failedServerRead) {
```

Comment:
Aside from the user not having read access in their policy, are there any other scenarios which would cause the version to not be read from the server? If so, my comments below regarding the alert banner warning aren't relevant. 😝

Reply:
Nope, it'd only be the policy missing (or an additional Sentinel policy on the path) that could prevent it.